src/components/WeekCalendar.tsx

```diff
@@ -63,101 +79,61 @@ const WeekCalendar = ({ campaigns, events, focusDate, onFocusDate }: WeekCalenda
 
     return (
         <>
-            <View position="absolute" right="15rem" top="-2.6rem">
-                <Flex alignItems="center">
-                    <ActionButton data-testid="back-button" onPress={
-                        () => onFocusDate(new Date(new Date(focusDate).setDate(focusDate.getDate() - 7)))
-                    }>
-                        <Msg id="misc.calendar.prev" />
-                    </ActionButton>
-                    <View data-testid="selected-date" padding="size-100">
-                        <FormattedDate
-                            day="2-digit"
-                            month="short"
-                            value={ calendarStartDate }
-                        />
-                    </View>
-                    <ActionButton data-testid="fwd-button" onPress={
-                        () => onFocusDate(new Date(new Date(focusDate).setDate(focusDate.getDate() + 7)))
-                    }>
-                        <Msg id="misc.calendar.next" />
-                    </ActionButton>
-                </Flex>
-            </View>
-            <div ref={ calendarWrapper } data-testid="calendar-wrapper" style={{
-                height: '100%',
-                overflow: 'scroll',
-                width: '100%',
-            }}>
-                <div style={{
-                    background: '#f5f5f5',
-                    display:'flex',
-                    gap:'1rem',
-                    justifyContent: 'space-between',
-                    position: 'sticky',
-                    top: 0,
-                    width: '100%',
-                    zIndex: 1,
-
-                }}>
+            <Box { ...{ ref: calendarWrapper } } data-testid="calendar-wrapper" height={ 1 } overflow="scroll" width={ 1 }>
```

Comment:
Correct me if I'm wrong (because it's quite difficult to read) but isn't there just a single top-level `<Box>` here? If that's the case, there should be no need to wrap in a fragment.